pkg/bucket/bucket.go

```diff
@@ -67,15 +69,31 @@ func CmdList() *cobra.Command {
 	return cmd
 }
 
-// RunCreate runs a CLI command
+// RunCreate runs a CLI command. The defualt backingstore will be used as the underlying storage for buckets created using the CLI
 func RunCreate(cmd *cobra.Command, args []string) {
 	log := util.Logger()
 	if len(args) != 1 || args[0] == "" {
 		log.Fatalf(`Missing expected arguments: <bucket-name> %s`, cmd.UsageString())
 	}
 	bucketName := args[0]
 	nbClient := system.GetNBClient()
-	err := nbClient.CreateBucketAPI(nb.CreateBucketParams{Name: bucketName})
+
+	bucketClass, err := bucketclass.GetDefaultBucketClass(options.Namespace)
+	if err != nil {
+		log.Fatal(fmt.Errorf("Failed to get defualt bucketcalss with error: %v", err))
+	}
+
+	if !util.KubeCheck(bucketClass) {
```

Comment:
is there a reason for calling again kubecheck for the bucketclass? it's called in getDefaultBucketclass

Reply:
also it checks readiness so I guess that's why you resolved the previous comment

Reply:
Right, I called getDefaultBucketclass per you last comment instead of doing it manually and forgot to remove this check